snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/japi/io/SnomedRefSetDSVExportTest.java

```diff
@@ -180,7 +181,14 @@ public void mapTypeDSVExport() throws Exception {
 
 	private Iterable<SnomedConstraint> getConstraints(SnomedConcepts concepts) {
 		return SnomedRequests
-				.prepareGetApplicablePredicates(MAIN_BRANCH, idsOf(concepts), ancestorsOf(concepts), Collections.emptySet()).getSync();
+				.prepareGetApplicablePredicates(MAIN_BRANCH, idsOf(concepts), ancestorsOf(concepts), Collections.emptySet(), relationshipsOf(concepts)).getSync();
+	}
+
+	private Set<String> relationshipsOf(SnomedConcepts concepts) {
+		return concepts.getItems()
+				.stream()
+				.flatMap(concept -> concept.getRelationships().stream().map(relationship -> relationship.getTypeId() + Ecl.EQUAL + relationship.getDestinationId()))
```

Comment:
The format of a relationship MRCM domain should be moved to the `SnomedConstraintDocument` class.
For example: `SnomedConstraintDocument.relationshipDomainOf(typeId, destinationId)`.